package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "http-signature",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
package.json and CHANGES.md say "1.3.2", but this says 1.3.1

Reply:
Hrm.. I guess I first built the cloned version before bumping the package version....